Libraries/libutil/Tests/test_FSUtil.cpp

```diff
@@ -83,3 +83,21 @@ TEST(FSUtil, ResolveRelativePath)
     EXPECT_EQ("c/a/b", FSUtil::ResolveRelativePath("a/b", "c"));
     EXPECT_EQ("/a/b", FSUtil::ResolveRelativePath("/a/b", "/c"));
 }
+
+TEST(FSUtil, IsFileExtension)
```

Comment:
Maybe I'm confused, but where was this added? Is the implementation in a later commit?